stats/opentelemetry/client_metrics.go

```diff
@@ -119,22 +134,46 @@ func (h *clientStatsHandler) streamInterceptor(ctx context.Context, desc *grpc.S
 	}
 
 	startTime := time.Now()
+	ctx, span := h.createCallTraceSpan(ctx, method)
```

Comment:
i think we should just not call createCallTraceSpan if tracing is disabled

Reply:
That's being handled inside createCallTraceSpan, in case tracing is disabled, it won't update the context and return empty span. This is similar pattern which was been followed for metrics. Shall we change this?